pkg/store/logger.go

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
why we need the N logger? We have already simple and file logger in logger package.